site/schemas/organisation.schema.ts

```diff
@@ -20,7 +20,7 @@ export const defaultModes: ModeType = {
 
 export const organisationSchema = z.object({
     name: z.string(setZodDefaultError("Enter an organisation name")).min(3),
-    adminAreaCodes: z.array(z.string()).min(1, { message: "Atleast 1 area code is required" }),
+    adminAreaCodes: z.array(z.string()).min(1, { message: "At least 1 area code is required" }),
     PK: z.string().optional(),
     mode: modeSchema.optional().default(defaultModes),
```

Comment:
Not sure why but the LoadingBox warning is showing up in this PR could the changes from main be pulled into this branch? Or could the LoadingBox import be corrected.

Reply:
fixed 👍